src/main/groovy/org/shipkit/internal/gradle/e2e/E2ETestingPlugin.java

```diff
@@ -47,12 +56,12 @@ void create(String gitHubRepoUrl) {
         }
 
         private CloneGitRepositoryTask createCloneProjectFromGitHub(String gitHubRepoUrl, String repoName) {
-            // TODO add depth clone configuration for shallow clone
             CloneGitRepositoryTask clone = project.getTasks().create(
                     "cloneProjectFromGitHub" + capitalize(repoName),
```

Comment:
I know that this was here before. I'm wondering if we should favor shorter task names in public API for cleaner output and better usability. Examples:
 - cloneProjectFromGitHubMockito -> cloneMockito
 - cloneProjectToWorkDir -> copyMockito